webpack.config.js

```diff
@@ -1,9 +1,19 @@
-const DtsBundleWebpack = require('dts-bundle-webpack');
 const webpackRxjsExternals = require('webpack-rxjs-externals');
 const path = require('path');
+var DeclarationBundlerPlugin = require('declaration-bundler-webpack-plugin');
 
 const IS_CI = !!process.env.CI;
 
+// NOTE: declaration-bundler-webpack-plugin problem fix https://github.com/TypeStrong/ts-loader/issues/263
+let buggyFunc = DeclarationBundlerPlugin.prototype.generateCombinedDeclaration;
```

Comment:
Hmmm... 
- DeclarationBundlerPlugin is 4 years old
- dts-bundle-webpack is 1 year old

What about ts-declaration-webpack-plugin?

Reply:
Used `DeclarationBundlerPlugin` because it recommended in `ts-loader` docs and most popular on npm.

Tried to use `ts-declaration-webpack-plugin` but it doesn't work for our setup. It just throws an error `src/Compass.d.ts not found`. Started to work after I removed `declaration` option from tsconfig, but just generating empty `d.ts` file.. and there are no too much config options.

Also tried `npm-dts-webpack-plugin` but it has some strange errors as well

Tried to revert `dts-bundle-webpack`, it actually works but the latest version has 1 low-level vulnerability.

So guess there is two options, stay with `dts-bundle-webpack` or leave 
current `DeclarationBundlerPlugin`. Doesn't see any reasonable options for now